source/codegen/service_helpers.py

```diff
@@ -62,6 +62,25 @@ def create_args_for_callback(parameters):
     ])
 
 
+def _is_array_requiring_conversion(parameter):
```

Comment:
Note I pulled out this helper because the control flow in `create_standard_arg` was getting totally nuts and we're losing track of what the point of the special cases are. So this collects several such cases into one and gives it a name.

The functional change here is using grpc_type to match `repeated nidevice_grpc.Session` instead of the pre-existing code that only matched `ViSession[]` for ivi drivers.

Testing done: all other drivers output the same code.

Reply:
Thanks for the cleanup! This is much easier to read through.